bin/stronghold-cli

```diff
@@ -166,13 +180,29 @@ opts.each do |opt, arg|
     end
     client.head.update({
         :path => $path,
-        :author => "command-line on #{Socket.gethostname}",
+        :author => "command-line on #{Socket.gethostname}, USER=#{ENV['USER']}",
         :data => intract(client.head.tree.peculiar($path), internal_path, to),
         :comment => comment || "At level #{$path} set #{arg}"
       })
+  when '--set-json'
+    mode_ify(opt)
+    json = JSON.parse(arg)
+    old_data = client.head.tree.peculiar($path)
+    new_data = intract_json(old_data, json)
+    unless old_data == new_data
+      client.head.update({
+        path: $path,
+        author: "command-line on #{Socket.gethostname}, USER=#{ENV['USER']}",
+        data: new_data,
+        comment: "At level #{$path} set from file #{arg}"
+      })
+    end
   when '--get'
     mode_ify(opt)
     puts format(extract(valid_tree(), data_path(arg)))
+  when '--get-json'
+    mode_ify(opt)
+    puts format(client.head.tree.peculiar($path))
```

Comment:
Actually I forgot that `--get` already returns JSON. Is this one returning the non-materialized view ?